lib/rester/rspec.rb

```diff
@@ -0,0 +1,57 @@
+require 'json'
+
+RSpec.configure do |config|
+  config.before :all, rester: // do |ex|
+    # Load the stub file
+    @stub_filepath = ex.class.metadata[:rester]
+    @stub = YAML.load_file(@stub_filepath)
+
+    # Hook up the LocalAdapter with the Service being tested
+    unless (klass = ex.class.described_class) < Rester::Service
+      raise "invalid service to test"
+    end
+    @adapter = Rester::Client::Adapters::LocalAdapter.new(klass, {})
+  end
```

Comment:
Where are these instance variables being stored? Is it possible we could have a conflict at some point?  Maybe we should use a `rester_` prefix?